src/test/java/Homework16.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public void RegistrationNavigation () {
+
+        WebDriver Driver = new ChromeDriver();
```

Comment:
For variables, the Java naming convention is to always start with a lowercase letter and then capitalize the first letter of every subsequent word.
For example WebDriver **driver** = new ChromeDriver();